load.php

```diff
@@ -23,6 +23,18 @@
 			'block-on-failing' => false,
 			'hide-column' => false,
 		],
+		'clone-republish' => [
+			'enabled' => true,
+			'post-types' => [
+				'post',
+				'page',
+			],
```

Comment:
Setting these here will mean we're hard coding to these 2 post types still. I'm thinking the default value should be `null` so it applies to all public post types by default.